controllers/mediaController.js

```diff
@@ -0,0 +1,60 @@
+// import Media schema from models
+const { User } = require('../models/mainModels.js');
+// destructured media property of User schema
+const { media } = User;
+
+
+// object contains all mediaController middleware
+const mediaController = {};
+
+
+// middleware to get entire media profile of user
+mediaController.getMedia = (req, res, next) => {
+  // find all media catalog
+  User.findOne({_id:req.query.userId}).exec()
+    .then(response => {
+      // console.log to see if fired
+      console.log("mediaController getMedia fired")
+      // store in local memory
+    res.locals.media = response;
+    return next();
+  })
```

Comment:
nitpicky, but can we tighten up indentation in this section? it's a little tough to follow the thread